package.json

```diff
@@ -18,22 +19,19 @@
   "files": [
     "index.js",
     "bin/*.js",
-    "lib/*.js",
-    "lib/commands/*.js",
-    "lib/instrumenters/*.js",
-    "!**/*covered.js"
+    "lib/**/*.js"
   ],
   "nyc": {
     "exclude": [
       "node_modules",
       "bin",
       "coverage",
+      "self-coverage",
       "test/fixtures/coverage.js",
       "test/build/*",
       "test/src/*",
       "test/nyc.js",
       "test/process-args.js",
```

Comment:
what's weird is that on master right now, if you just have `test` in the `exclude` list the test fails... i have no idea why but I would love to see that change work

Reply:
I think this is because we need to allow most of `test/fixtures/**`.